docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,16 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is kind of query known as a "non-scoring" query. It does not give a score,
```

Comment:
This is correct, but could you make it clearer from the beginning that a "filter" is a normal query, just one that doesn't score? You already say this, but in the current order the definition isn't as clear as it could be. I think this just needs a little rephrasing.

Reply:
also: "is a kind" if you leave that part in anywhere.